### PR TITLE
tests(invite service): add tests for empty string search

### LINKED ISSUE #8491: [Bug] Invite page crashed admin panel

### PR DESCRIPTION
**Pull request by aditya-mitra**
## References

closes #8491 

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

File: packages/server-core/src/social/invite/invite.class.ts [hunk 1]
```diff
@@ -199,6 +199,8 @@ export const inviteAll = async (inviteService: Invite, query: Query, user: UserT
   }
   if (!query.existenceCheck) delete query.userId
   delete query.existenceCheck
+  delete query.search
```
Comment (hanzlamateen) [inline packages/server-core/src/social/invite/invite.class.ts:202]: Maybe its good to have a check here like

```
if (query.search) {
  delete query.search
}
```

**Review by hanzlamateen — CHANGES_REQUESTED** [review]

File: packages/server-core/src/social/invite/invite.class.ts (hunk 1 above)
Comment (HexaField) [reply packages/server-core/src/social/invite/invite.class.ts:202]: yes

Comment (aditya-mitra) [reply packages/server-core/src/social/invite/invite.class.ts:202]: Actually it was the reason for the bug
image

Comment (hanzlamateen) [reply packages/server-core/src/social/invite/invite.class.ts]: Yes, it can be removed. But just put that delete in an if condition. I think as you merged the PR with latest dev that change is no longer required?

Comment (aditya-mitra) [reply packages/server-core/src/social/invite/invite.class.ts]: Yes, you are right
That code is now present in latest dev
Maybe we could add the tests in this PR only

Comment (aditya-mitra) [reply packages/server-core/src/social/invite/invite.class.ts]: Yes, you are right
That code is now present in latest dev
Maybe we could add the tests in this PR only

**Review by hanzlamateen — APPROVED** [review]
